bin/ci

```diff
@@ -7,16 +7,14 @@ declare main_branch="main"
 
 # Generate a coma-separated list of quoted app names that changed in this PR
 apps=$(
-  git whatchanged --name-only --pretty="" "origin/${main_branch}..${cursor}" | \
-  egrep "^apps" | \
-  sed "s/^apps\/\(.*\)\/.*$/\"\1\"/g" | \
+  git whatchanged --name-only --pretty="" "origin/${main_branch}..${cursor}" apps | \
+  awk -F '/' '{print "\"" $2 "\""}' | \
   sort -u | \
-  tr "\n" "," | \
-  sed "s/,$//"
+  paste -sd ','
 )
 
 # Patch the config template to generate a custom workflow that tests only
 # changed apps
 yq eval \
-  ".workflows.arnold.jobs.[3].test-bootstrap-app.matrix.parameters.app = [${apps}]" \
+  "select(.workflows.arnold.jobs.[].test-bootstrap-app.matrix.parameters.app=[${apps}])" \
```

Comment:
Hat tip to @madmatah :muscle:

Reply:
![](https://media.giphy.com/media/B37cYPCruqwwg/giphy.gif)